components/camel-sip/src/main/java/org/apache/camel/component/sip/SipConsumer.java

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
this appears to be a rename from SipSubscriber.java, its a shame Github doesn't show it more concisely though